lib/document-to-html.js

```diff
@@ -100,21 +100,55 @@ function DocumentConversion(options, comments) {
             return [];
         }
     }
+
+    function enrichListItemAttrs(num, li) {
+        if(li==null && li.tag==null || li.tag.tagName!=='li') return;
```

Comment:
Maybe add some grouping - so it's more readable (without noticing that we have `&&` then `||`.
```suggestion
        if ((li == null && li.tag == null) || li.tag.tagName !== "li") return;
```